packages/tasks/src/tasks/text-generation/data.ts

```diff
@@ -51,32 +55,32 @@ const taskData: TaskDataCustom = {
 	],
 	models: [
 		{
-			description: "A large language model trained for text generation.",
-			id: "bigscience/bloom-560m",
+			description: "A large text-generation model trained to follow instructions.",
+			id: "google/gemma-2-2b-it",
 		},
 		{
 			description: "A large code generation model that can generate code in 80+ languages.",
 			id: "bigcode/starcoder",
 		},
 		{
-			description: "A very powerful text generation model.",
-			id: "mistralai/Mixtral-8x7B-Instruct-v0.1",
+			description: "A very powerful text generation model trained to follow instructions.",
+			id: "meta-llama/Meta-Llama-3.1-8B-Instruct",
```

Comment:
```suggestion
			id: "meta-llama/Meta-Llama-3.1-405B-Instruct",
```

For the LOLs, it technically is the most powerful open text generation model hehe.

Reply:
yeah makes sense!